README.md

```diff
@@ -3,17 +3,27 @@
 This example demonstrates how to use the Blob Storage service with Go. If you don't have a Microsoft Azure subscription you can get a FREE trial account [here](https://azure.microsoft.com/pricing/free-trial).
 
 ##Instructions
+
+###Using Azure Storage Emulator
+1. Download, install and run the [Azure Storage Emulator](https://azure.microsoft.com/documentation/articles/storage-use-emulator/).
+2. Get the [Azure SDK for Go](https://github.com/Azure/azure-sdk-for-go) using command `go get -u github.com/Azure/azure-sdk-for-go`
+3. Get this sample using command `go get -u github.com/Azure-Samples/storage-blob-go-getting-started`
+4. Compile and run the sample.
+
+###Using Storage Service
 1. Create a [storage account](https://azure.microsoft.com/documentation/articles/storage-create-storage-account/#create-a-storage-account) through the Azure Portal.
 2. Set environment variables `AZURE_STORAGE_ACCOUNT_NAME = <ACCOUNT_NAME>` and `AZURE_STORAGE_ACCOUNT_KEY = <ACCOUNT_KEY>`.
 3. Get the [Azure SDK for Go](https://github.com/Azure/azure-sdk-for-go) using command `go get -u github.com/Azure/azure-sdk-for-go`
 4. Get this sample using command `go get -u github.com/Azure-Samples/storage-blob-go-getting-started`
-5. Compile and run the sample.
+5. Open the storageExample.go file and replace this line `credentials, err := getCredentials(emulator)` with this line `credentials, err := getCredentials(account)`, inside the blobSamples function.

```

Comment:
> err := getCredentials(emulator) [](start = 70, length = 32)

I'd recommend also keeping a line "credentials, err := getCredentials(account)" with appropriate comments sot hat easy to uncomment line and use sample against service.